.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
remove useless comments please

Reply:
@Marwyn42 and I disagree about the usefulness of this comment.